backend/templates/inputs/model-na-31-2.json

```
@@ -49,11 +64,15 @@
     "political_group_votes": [
       {
         "number": 1,
-        "total": 0,
+        "total": 30,
```

```suggestion
        "total": 0,
```
These should be 0 I think, as all values above are also 0.

---

I've updated the voters and votes counts to reflect these, so that we don't just have all zeroes in this example.